VMware.CloudFoundation.PasswordManagement.psm1

```diff
@@ -1345,7 +1345,13 @@ Function Get-PasswordPolicyDefault {
     $sddcManagerPasswordExpiration | Add-Member -notepropertyname 'minDays' -notepropertyvalue "0"
     $sddcManagerPasswordExpiration | Add-Member -notepropertyname 'warningDays' -notepropertyvalue "7"
     $sddcManagerPasswordComplexity = New-Object -TypeName psobject
-    $sddcManagerPasswordComplexity | Add-Member -notepropertyname 'minLength' -notepropertyvalue "8"
+    if ($version -ge "5.2") {
+        $sddcManagerPasswordComplexity | Add-Member -notepropertyname 'minLength' -notepropertyvalue "15"
+    } elseif ($version -ge "5.1") {
+        $sddcManagerPasswordComplexity | Add-Member -notepropertyname 'minLength' -notepropertyvalue "12"
+    } else {
+        $sddcManagerPasswordComplexity | Add-Member -notepropertyname 'minLength' -notepropertyvalue "8"
+    }
```

Comment:
```suggestion
switch ($version) {
    {$_ -ge "5.2"} {
        $sddcManagerPasswordComplexity | Add-Member -notepropertyname 'minLength' -notepropertyvalue "15"
    }
    {$_ -ge "5.1"} {
        $sddcManagerPasswordComplexity | Add-Member -notepropertyname 'minLength' -notepropertyvalue "12"
    }
    default {
        $sddcManagerPasswordComplexity | Add-Member -notepropertyname 'minLength' -notepropertyvalue "8"
    }
}
```

This may be more efficient. Please test.

Reply:
Will leave as-is.